### PR TITLE
Add `remove_selections` command

### LINKED ISSUE #1062: Add `remove_selections` (Remove selections matching the regex) key
#### Describe your feature request

We are currently have:

| Key | Description | Command |
| -- | -- | -- |
| <kbd>K</kbd> | Keep selections matching the regex | `keep_selections` |

but there is no `remove_selections` (Remove selections matching the regex) key which i found somewhat useful.

It's there a particular reason for excluding it or it's just been forgotten?

### PR DESCRIPTION
**Pull request by Omnikar**
Closes #1062.

I have added the keybind to `A-K`, let me know if it should be changed.

### CODE REVIEW

**Comment by pickfire** [comment]
Can you please update the docs for this?

**Review by pickfire — APPROVED** [review]
Looks good to me but just missing docs change.

**Comment by Omnikar** [comment]
Ah yeah, I keep forgetting to update the docs when I add a new command.

File: helix-term/src/keymap.rs [hunk 1]
```diff
@@ -556,6 +556,7 @@ impl Default for Keymaps {
             "=" => format_selections,
             "J" => join_selections,
             "K" => keep_selections,
+            "A-K" => remove_selections,
             // TODO: and another method for inverse
```
Comment (archseer) [inline helix-term/src/keymap.rs:560]:
```suggestion
```

**Pull request merged**